### PR TITLE
add wordCounter work on bugs

### PR DESCRIPTION
**Pull request by PavelBoPavel**

### CODE REVIEW

File: homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    @Before
+    public void setup(){
+        uniqueWordCounter = new UniqueWordCounter();
+        addInitialWords();
+    }
+
+    @Test
+    public void addWord_test(){
+        addInitialWords();
```
Comment (KaydunovDenis) [inline homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java:21]: You use it method in method: @Before setup

File: homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    private void addInitialWords() {
+        uniqueWordCounter.addWord("hello");
+        uniqueWordCounter.addWord("dog");
+        uniqueWordCounter.addWord("cat");
+        uniqueWordCounter.addWord("hello");
+        uniqueWordCounter.addWord("hello");
+        uniqueWordCounter.addWord("dog");
+    }
```
Comment (KaydunovDenis) [inline homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java:43]: I think it method me by in class UniqueWordCounter.

**Review by KaydunovDenis — CHANGES_REQUESTED** [review]
Missing the description provided.

File: homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java (hunk 1 above)
Comment (max-java) [reply homework/PavelBo/collections/wordCounter/src/test/java/com/tutrit/java/quickstart/UniqueWordCounterTest.java:21]: therefore second call in test method is redundant.

**Review by max-java — APPROVED** [review]
minor comments, otherwise lgfm

**Review by KaydunovDenis — APPROVED** [review]

**Review by VasiliZas — APPROVED** [review]

**Review by max-java — APPROVED** [review]
ok

**Pull request merged**